docs/virtualization_troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting virtualization backup and restore
+
```

Comment:
Perhaps a subheading that signify this is a typical usecase that should pass without errors.